.golangci-kal.yml

```diff
@@ -17,13 +17,12 @@ linters:
               - "conditions" # Ensure conditions have the correct json tags and markers.
               - "conflictingmarkers"
               - "duplicatemarkers" # Ensure there are no exact duplicate markers. for types and fields.
-              #- "forbiddenmarkers" # Ensure that types and fields do not contain any markers that are forbidden.
-              # - "integers" # Ensure only int32 and int64 are used for integers.
+              - "integers" # Ensure only int32 and int64 are used for integers.
               - "jsontags" # Ensure every field has a json tag.
               # - "maxlength" # Ensure all strings and arrays have maximum lengths/maximum items.
               # - "nobools" # Bools do not evolve over time, should use enums instead.
-              # - "nodurations" # Prevents usage of `Duration` types.
-              # - "nofloats" # Ensure floats are not used.
+              - "nodurations" # Prevents usage of `Duration` types.
```

Comment:
We have `Duration` in the following:

- https://github.com/kubernetes-sigs/kueue/blob/7b9f971745eb463c5940acca2d131d7114b2cfda/apis/config/v1beta1/configuration_types.go#L219
- https://github.com/kubernetes-sigs/kueue/blob/7b9f971745eb463c5940acca2d131d7114b2cfda/apis/config/v1beta1/configuration_types.go#L234

But, I agree with the removal of Duration usage in v1beta2.

Reply:
hmmm, good spot but in that case, is the build going to fail due to enabling?

Reply:
Right now, we aren't actually running this on config api.

We could do that but configs are not necessarly CRDs so I'm not sure if we are trying to make our config api Kubernetes API compatible.

Reply:
Also, I'm wondering if we could enable the linter, but just comment on a specific usage as legacy use.